VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+* **Track record:** Delegators will likely look at the track record of the validators they plan to delegate to. This includes seniority, past votes on proposals, historical average uptime and how often the node was compromised.
+
+Apart from these objective criteria that will be displayed in the Tendermint wallet’s UI, there will be a possibility for validators to signal a website address to complete their resume. Validators will need to build reputation one way or another to attract delegators. For example, it would be a good practice for validators to have their setup audited by third parties. Note though that the Tendermint company will not approve or conduct any audit itself.
+
+### Do validators need to be publicly identified?
+
+No, they don’t. Each delegator will value each validator based on objective and subjective criteria. Validators will be able (and are recommended) to register a website address when they nominate themselves so that they can advertise their operation as they see fit. Some delegators may prefer a website that clearly displays the team running the validator and their resume, while others might prefer anonymous validators with positive track records. Most likely both identified and anonymous validators will coexist in the validator set.
+
+### What are the directives of a validator?
+
+Validators have two main directives:
+* **Be able to constantly run a correct version of the software:** validators need to make sure that their servers are always online and their private keys are not compromised. 
+* **Actively participate in governance:** validators are required to vote on every proposal.
+
+Additionally, validators are expected to be active members of the community. They should always be up to date with the current state of the ecosystem so that they can easily adapt to any change.
+
+### What does ‘participate in governance’ entail?
+
+Validators and delegators on the Cosmos Hub can vote on proposals to change preset parameters of the system automatically (such as the block gas limit), coordinate upgrades, as well as vote on amendments to the human-readable constitution that govern the policies of the Cosmos Hub. The governance system allows for smooth coordination within the network. 
+
+Validators play a special role in the governance system. Being the pillars of the system, they are required to vote on every proposal. It is especially important since delegators who do not vote will inherit the vote of their validators. If a validator does not vote on a proposal it will get a small punishment, and if it does not vote on many successive proposals, the validator will become unbonded.
```

Comment:
Are you sure about unbonding a validator that fails to vote even though he is signing precommits? 

I don't think we should unbond a validator for that reason. However on the other hand those small punishments can really add up otherwise.

Reply:
It's true we don't need to unbond anymore since if they're inactive and their precommits are not included they will get unbonded anyway.

Reply:
@adrianbrink signing pre-commits is a kind of vote this is what I meant. - what votes were you talking about

Reply:
Okay looks addressed 👍